src/pages/en/core-concepts/routing.md

```diff
@@ -187,6 +165,55 @@ const { title, text } = Astro.props;
 </html>
 ```
 
+### Server (SSR) Mode
+In [SSR mode](/en/guides/server-side-rendering/), dynamic routes are defined the same way: include `[param]` or `[...path]` brackets in your file names to match arbitrary strings or paths. But because the routes are no longer built ahead of time, the page will be served to any matching route, and `getStaticPaths` has no effect. 
```

Comment:
```suggestion
In [SSR mode](/en/guides/server-side-rendering/), dynamic routes are defined the same way: include `[param]` or `[...path]` brackets in your file names to match arbitrary strings or paths. But because the routes are no longer built ahead of time, the page will be served to any matching route. Since there are no "static" routes, `getStaticPaths` has no effect and is not necessary.
```
Not married to this wording, but it seems to me that there are two nuances I might choose to address here:

- "gSP has no effect" sounds a little weirder and less explicit than "so you don't NEED/USE gSP to make your dynamic routes"
- you could totally use the name itself to emphasize the difference/lack of need to use it! 

I leave it to you to decide! (But I did think "has no effect" was a bit weird sounding.)

Reply:
Agreed, love the idea of referencing the name! I wanted to make it clear that you _shouldn't_ use GSP:

> Since these are not _static_ routes, `getStaticPaths` should not be used.